From c88b0fd50f7f007600725a7a04452772dee5e208 Mon Sep 17 00:00:00 2001
From: Praveen Paneri <praveen.paneri@intel.com>
Date: Fri, 26 Jun 2015 13:50:45 +0530
Subject: [PATCH] SQUASHME! [VPG]: drm/i915: Change permissions for
 i915_videostatus node

The permissions shouldn't have been granted to others and should
have been granted only to the user.

SQUASHME! - This patch should be squashed into the following existing patch:
Author: sudheer gurram <sudheer.gurram@intel.com>
Change-Id: I30a18d01f5735546c80ac5660b45c0419771ed98
MUST_REBASE [VPG]: drm/i915: Fix for the gamma curve issue

Following patch enables DPST in video playback scenarios based on a setting,
which is passed from user mode process(CoreU)

Tracked-On: https://jira01.devtools.intel.com/browse/OAM-2284
Change-Id: I23f6e66bf186ad894df9d616e57c03bea09f3113
Signed-off-by: Praveen Paneri <praveen.paneri@intel.com>
---
 drivers/gpu/drm/i915/i915_sysfs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/i915_sysfs.c b/drivers/gpu/drm/i915/i915_sysfs.c
index 836cb24..d047f8b 100644
--- a/drivers/gpu/drm/i915/i915_sysfs.c
+++ b/drivers/gpu/drm/i915/i915_sysfs.c
@@ -239,7 +239,7 @@ store_i915_videostatus(struct device *kdev, struct device_attribute *attr,
 	return n;
 }
 
-static DEVICE_ATTR(i915_videostatus, (S_IRUGO | S_IWOTH),
+static DEVICE_ATTR(i915_videostatus, (S_IRUGO | S_IWUSR),
 		show_i915_videostatus, store_i915_videostatus);
 static struct attribute *i915_videostatus_attrs[] = {
 	&dev_attr_i915_videostatus.attr,
-- 
1.9.1

